### ISSUE TITLE
download() function + Selenoid: HTTP Status: '404' when other tab was opened before using the function

### ISSUE THREAD

**Issue by abzhasso** [selenide/selenide#929]
## The problem
download() function doesn't work with Selenoid if the other tab was opened before using it.

## Details
download() function doesn't work if the other tab was opened before using it.
Test crushes with such logs:
```
апр 16, 2019 1:03:16 AM org.openqa.selenium.remote.ErrorCodes toStatus
INFO: HTTP Status: '404' -> incorrect JSON status mapping for 'unknown error' (500 expected)
апр 16, 2019 1:03:16 AM org.openqa.selenium.remote.ErrorCodes toStatus
INFO: HTTP Status: '404' -> incorrect JSON status mapping for 'unknown error' (500 expected)
...
```
I tested some variants and found that it persists with my environment with Selenoid.
I tried to use without Selenoid, with Firefox on Win10 - worked!
Maybe it's Selenoid issue. Please check this.

## Tell us about your environment
* **Selenide Version:**
5.2.2

* **Chrome\Firefox\IE Version:**
IE 11

* **Browser Driver Version:**
3.14.0 (latest from https://www.seleniumhq.org/download/ )

* **Selenium Version:** -

* **OS Version:**
Windows 10 + Windows Server 2012 (remote machine)
I use Selenoid, the latest (tried to update today - doesn't work too).

## Code To Reproduce Issue [ Good To Have ]
```
public class MyClass extends MyBaseTestClass {
    @Test
    public void MyFunction() {
        open("https://bus.gov.ru/pub/registry");
        $(byAttribute("ng-model", "form.agency")).setValue("13-Й ОТРЯД ПРОТИВОПОЖАРНОЙ СЛУЖБЫ НОВГОРОДСКОЙ ОБЛАСТИ");
        $(byText("Показать")).click();
        $(".result__title").shouldHave(exactText("13-Й ОТРЯД ПРОТИВОПОЖАРНОЙ СЛУЖБЫ НОВГОРОДСКОЙ ОБЛАСТИ")).scrollTo();
        $(".result__title").hover();
        $(byText("Подробная информация")).click();
        getWebDriver().close(); // use or not use - no difference

        switchTo().window("Карточка государственного (муниципального) учреждения");
        String s = getWebDriver().getTitle();
       // I'm just showing that it swithes correctly
        assert s.equals("Карточка государственного (муниципального) учреждения") : "error";

        $(byText("Документы (15)")).click();
        SelenideElement el = $(byText("ЕГРЮЛ"));
        try {
       // doesn't work, it works only if I open "Подробная информация" by direct link, very strange
            el.download();
        } catch (FileNotFoundException e) {
            e.printStackTrace();
        }
    }
}
```

**Comment by abzhasso** [comment]
As a workaround:
```
        String link = $(byLinkText("Подробная информация")).getAttribute("href");
        open(link);
```

**Comment by abzhasso** [comment]
I noticed that sendKeys method doesn't work anymore. It is very slow and also may throw "INFO: HTTP Status: '404' -> incorrect JSON status mapping for 'unknown error' (500 expected)". But it seems It was Selenium issue. I downloaded the new version of IEDriverServer.exe from here - https://github.com/SeleniumHQ/selenium/blob/master/cpp/prebuilt/Win32/Release/IEDriverServer.exe
and it fixed my problem!!! It is very strange that no one version work from this page: https://www.seleniumhq.org/download/
